src/main/java/org/gitlab4j/api/ServicesApi.java

```diff
@@ -493,4 +494,58 @@ public void deleteCustomIssueTrackerService(Object projectIdOrPath) throws GitLa
         delete(Response.Status.OK, null, "projects", getProjectIdOrPath(projectIdOrPath), "services", "custom-issue-tracker");
 
     }
+
+    /**
+     * Get the Custom Issue Tracker service settings for a project.
+     *
+     * <pre><code>GitLab Endpoint: GET /projects/:id/services/custom_issue_tracker</code></pre>
+     *
+     * @param projectIdOrPath id, path of the project, or a Project instance holding the project ID or path
+     * @return a ExternalWikiService instance holding the External Wiki service settings
+     * @throws GitLabApiException if any exception occurs
+     */
+    public EmailOnPushService getEmailOnPushService(Object projectIdOrPath) throws GitLabApiException {
+        Response response = this.get(Response.Status.OK, null, "projects", getProjectIdOrPath(projectIdOrPath), "services", "emails-on-push");
+        return (response.readEntity(EmailOnPushService.class));
+    }
+
+    /**
+     * Updates the Custom Issue Tracker service settings for a project.
+     *
+     * <pre><code>GitLab Endpoint: PUT /projects/:id/services/custom_issue_tracker</code></pre>
+     *
+     * The following properties on the CustomIssueTrackerService instance are utilized in the update of the settings:
+     * <p>
+     * description (optional), description
+     * issuesUrl (required), issue url
+     * newIssueUrl (required), new Issue url
+     * projectUrl (required), project url
+     * pushEvents (optional) - Enable notifications for push events
+     * title (optional), the title for the custom issue tracker
+     * </p>
+     *
+     * @param projectIdOrPath id, path of the project, or a Project instance holding the project ID or path
+     * @param customIssueTracker the CustomIssueTrackerService instance holding the settings
+     * @return a CustomIssueTrackerService instance holding the newly updated settings
+     * @throws GitLabApiException if any exception occurs
+     */
+    public EmailOnPushService updateEmailOnPushService(Object projectIdOrPath,  EmailOnPushService customIssueTracker) throws GitLabApiException {
+        GitLabApiForm formData = customIssueTracker.servicePropertiesForm();
+        Response response = put(Response.Status.OK, formData.asMap(), "projects", getProjectIdOrPath(projectIdOrPath), "services", "emails-on-push");
+        return (response.readEntity(EmailOnPushService.class));
+    }
+
+    /**
+     * Deletes the Custom Issue Tracker service for a project.
```

Comment:
Can you update javadoc accordingly to gitlab documentation? https://docs.gitlab.com/ee/api/services.html#emails-on-push